src/protocol/spi/cyrf6936.c

```diff
@@ -313,6 +313,7 @@ void CYRF_FindBestChannels(u8 *channels, u8 len, u8 minspace, u8 min, u8 max)
     CYRF_ConfigCRCSeed(0x0000);
     CYRF_SetTxRxMode(RX_EN);
     //Wait for pre-amp to switch from send to receive
+    CLOCK_ResetWatchdog();
     Delay(1000);
```

Comment:
Something is wrong here.  The watchdog will fire after 3 seconds, and it is reset in the event loop.  If it resets here, something is not working properly.  This needs to be removed and the root cause found before this patch can be accepted.

Reply:
fixed